Calculator.py

```diff
@@ -1,5 +1,5 @@
 def gather_inputs():
-    operation = input("Please type in operation you would like to complete: +, -, *, or /")
+    operation = input("Please type in operation you would like to complete: +, -, *, /, or ^")
```

Comment:
Can we have an exponentiation operator as this is the desired capability for this feature request?